content/en/agent/cluster_agent/admission_controller.md

```diff
@@ -0,0 +1,92 @@
+---
+title: Datadog Admission Controller
+kind: documentation
+further_reading:
+- link: "/agent/cluster_agent/clusterchecks/"
+  tag: "Documentation"
+  text: "Running Cluster Checks with Autodiscovery"
+- link: "/agent/cluster_agent/troubleshooting/"
+  tag: "Documentation"
+  text: "Troubleshooting the Datadog Cluster Agent"
+---
+
+## Overview
+The Datadog admission controller is a component of the Datadog Cluster Agent. The main benefit of the admission controller is to simplify the user's application pod configuration. For that, It has two main functionalities:
```

Comment:
Let's also link to Kubernetes Admission Controller guide (https://kubernetes.io/blog/2019/03/21/a-guide-to-kubernetes-admission-controllers/) and also mention that the Datadog's controller is of type MutatingAdmissionWebhook 
(https://kubernetes.io/docs/reference/access-authn-authz/admission-controllers/#mutatingadmissionwebhook)